src/Application/src/Calculator.cpp

```diff
@@ -0,0 +1,246 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file Calculator.cpp
+* @brief Function definitions and class for calculator
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include "Calculator.h"
+#include <stdlib.h>
+#include <iostream>
+#include <typeinfo>
+#include "CalculatorMessages.h"
+#include <sstream>
+namespace calculator{
+
+    enum class CalculateStates 
+    {
+	FindingNumber1,
+	OperatorFound,
+	FindingNumber2
+    };
+
+
+//*************/
+// Calculator public methods /
+///
+   
+    std::string fullEquation;
+
+    void Calculator::runCalculator()
+    {
+        std::cout << CalculatorMessages::INTRODUCTION_MESSAGE << std::endl;
+        std::cout << CalculatorMessages::INSTRUCTIONS_MESSAGE << std::endl;
+
+        std::string input;
+
+        getline(std::cin, input);
+
+        if (input == "")
+        {
+            getline(std::cin, input);
+        }
+        
+        if (validateInput(input))
+        {
+            char operation;
+            float number1;
+            float number2;
+
+            std::stringstream ss {fullEquation};
+            ss >> number1 >> operation >> number2;
+
+            std::cout << fullEquation << CalculatorMessages::EQUALS << calculate(operation, number1, number2) << std::endl;
+        }
+        else
+        {
+            std::cout << CalculatorMessages::ERROR_MESSAGE << CalculatorMessages::ERROR_MESSAGE_INVALID_INPUT << std::endl;
+        }
+
+        
+    }
+
+    bool Calculator::validateInput(std::string input)
+    {
+        CalculateStates CurrentState = CalculateStates::FindingNumber1;
+        char operation;
+        fullEquation = "";
+        std::string number2Str;
+        
```

Comment:
create a variable `bool isValid = true;`

then instead of `return false;` throughout the function do `isValid = false` and at the end of the function do `return isValid;`

Functions should ideally only have 1 `return`